frontend/test/metabase-db/mongo/line.cy.spec.js

```diff
@@ -0,0 +1,45 @@
+import { restore, addMongoDatabase, popover } from "__support__/e2e/cypress";
+
+const MONGO_DB_NAME = "QA Mongo4";
+
+// Skipping the whole describe block because it contains only one skipped test so far!
+// We don't want to run the whole beforeEeach block in CI only to skip the test afterwards.
+// IMPORTANT: when #16170 gets fixed, unskip both describe block and the test itself!
+describe.skip("mongodb > visualization > line chart", () => {
```

Comment:
This speeds up CI a little bit because we don't go through the effort of adding a database for nothing. Even if the test is skipped, `beforeEach()` block will execute.